pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/writer/impl/FixedByteChunkSVForwardIndexWriter.java

```diff
@@ -112,4 +113,12 @@ private void flushChunkIfNeeded() {
       writeChunk();
     }
   }
+
+  private static int normalizeChunkSize(int version, int numDocsPerChunk) {
```

Comment:
(minor) `normalizeDocsPerChunk` to be more clear?

Reply:
Absolutely, that's a better name.